pro/src/screens/OfferIndividual/StocksEvent/__specs__/StockEvent.edition.spec.tsx

```diff
@@ -0,0 +1,207 @@
+  contextValue,
+}: {
+  props: IStocksEventProps
+  storeOverride: Partial<RootState>
+  contextValue: IOfferIndividualContext
+}) => {
+  const store = configureTestStore(storeOverride)
+  return render(
+    <Provider store={store}>
+      <MemoryRouter initialEntries={['/creation/stocks']}>
+        <Route path="/creation/stocks">
+          <OfferIndividualContext.Provider value={contextValue}>
+            <StocksEvent {...props} />
+          </OfferIndividualContext.Provider>
+        </Route>
+        <Route path="/offre/:offer_id/v3/creation/individuelle/recapitulatif">
+          <div>Next page</div>
+        </Route>
+        <Route path="/offre/:offer_id/v3/creation/individuelle/stocks">
+          <div>Save draft page</div>
+        </Route>
+        <Route path="/offre/:offer_id/v3/creation/individuelle/informations">
+          <div>Previous page</div>
+        </Route>
+        <Notification />
+      </MemoryRouter>
+    </Provider>
+  )
+}
+
+describe('screens:StocksEvent:Edition', () => {
+  let props: IStocksEventProps
+  let storeOverride: Partial<RootState>
+  let contextValue: IOfferIndividualContext
+  let offer: Partial<IOfferIndividual>
+  let defaultStock: Partial<IOfferIndividualStock>
+
+  beforeEach(() => {
+    defaultStock = {
+      id: 'STOCK_ID',
+      quantity: 10,
+      price: 10.01,
+      remainingQuantity: 6,
+      bookingsQuantity: 4,
+      isEventDeletable: true,
+    }
+    offer = {
+      id: 'OFFER_ID',
+      lastProvider: null,
+      venue: {
+        departmentCode: '75',
+      } as IOfferIndividualVenue,
+      stocks: [defaultStock as IOfferIndividualStock],
+    }
+    props = {
+      offer: offer as IOfferIndividual,
+    }
+    storeOverride = {}
+    contextValue = {
+      offerId: null,
+      offer: null,
+      venueList: [],
+      offererNames: [],
+      categories: [],
+      subCategories: [],
+      setOffer: () => {},
+    }
+    jest
+      .spyOn(api, 'getOffer')
+      .mockResolvedValue({} as GetIndividualOfferResponseModel)
+  })
+
+  it('should allow user to delete a stock', async () => {
+    jest.spyOn(api, 'deleteStock').mockResolvedValue({ id: 'OFFER_ID' })
+    renderStockEventScreen({
+      props,
+      storeOverride,
+      contextValue,
+    })
+    await userEvent.click(screen.getByTestId('stock-form-actions-button-open'))
+    await userEvent.click(screen.getByText('Supprimer le stock'))
+    expect(
+      screen.getByText('Voulez-vous supprimer ce stock ?')
+    ).toBeInTheDocument()
+    expect(
+      screen.getByText(
+        'L’ensemble des utilisateurs concernés sera automatiquement averti par e-mail.'
+      )
+    ).toBeInTheDocument()
+    await userEvent.click(
+      await screen.findByText('Supprimer', { selector: 'button' })
+    )
+    expect(screen.getByText('Le stock a été supprimé.')).toBeInTheDocument()
+    expect(api.deleteStock).toHaveBeenCalledTimes(1)
+    expect(api.deleteStock).toHaveBeenCalledWith('STOCK_ID')
```

Comment:
```suggestion
    expect(api.deleteStock).toHaveBeenCalledWith('STOCK_ID')
    expect(api.deleteStock).toHaveBeenCalledTimes(1)
```
je trouve qu'on à plus d'info en mettant le `calledWith` avant. C'est frustrant d'avoir un erreur "mauvais nombre de call puis apres fix une autre: "voila ce qui casse dans le call"